File: C005_CLDemo.py
#!/usr/bin/env python36

import sys;

print ("You entered ", (len(sys.argv)-1), " command-line arguments")

if ( len(sys.argv) > 1 ):
  print ("They were:")
  for i in range(len(sys.argv)-1):
    print ("   " + sys.argv[i+1])

File: C005_CLDemo.js
console.log("You entered " + (process.argv.length-2) + " command-line arguments");

if (process.argv.length > 2) {
  console.log("They were:");
  for (i=0; i<process.argv.length-2; i++) {
    console.log("   " + process.argv[i+2]);
  }
}
